webserver/script/views.py

```diff
@@ -27,6 +30,16 @@ def post(self, request, format=None):
         sca_response = {"controlled_load" : getScaData(item_controlled), "uncontrolled_load" : getScaData(item_uncontrolled)}
         return Response(json.dumps(sca_response))
 
+class CostBenefitAnalysisRunner(APIView):
+    def post(self, request, format=None):
+        # TODO: requires an updated version of CBA Tool 
+        # While waiting for the updated CBA tool,
+        # this setup runs the split file module correctly, but saves the csv to test files in CBA tool
+        # CBA tool instead is processing old ev load profiles in the meantime and saving basecase results to db
+        split_file(county = request.data['county'])
+        ModelInstance()
```

Comment:
It's been a bit hard to follow, but what is this ModelInstance instantiation doing?

Reply:
ModelInstance() is how to call the cba_tool to run. It takes in the csv's in EV loads/ load profiles, runs various algorithms to figure out the cost benefit analysis and then populates the results directory with new csv's containing the results. UploadToPostgres then inserts these results to the database.

Reply:
I can hop on a quick zoom call tomorrow to do a walk through if needed. I know it's a lot in this PR. Thanks